doc/source/cost.rst

```diff
@@ -0,0 +1,45 @@
+Estimating costs
+================
+
+It is difficult to estimate costs exactly, because this depends highly on your
```

Comment:
Cost estimates depend highly on your deployment setup. Several factors that significantly influence cost estimates, include:

- Hardware provided to users
- Number of users
- Usage patterns of users